vulnerablecode/urls.py

```diff
@@ -27,13 +27,17 @@
 from rest_framework.routers import DefaultRouter
 
 from vulnerabilities.api import PackageViewSet
+from vulnerabilities.api import VulnerabilityViewSet
 
 
-api_router = DefaultRouter()
-api_router.register(r'packages', PackageViewSet)
+api_router = DefaultRouter(trailing_slash=False)
+api_router.register(r"packages", PackageViewSet)
+# `DefaultRouter` requires `basename` when registering viewsets which don't
+# define a queryset.
+api_router.register(r"vulnerabilities", VulnerabilityViewSet, basename="vulnerability")
 
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    re_path(r'^api/', include(api_router.urls))
+    path("admin/", admin.site.urls),
+    re_path(r"^api/", include(api_router.urls)),
```

Comment:
What is the need for `re_path` instead of `path` here?

Reply:
I haven't touched that, that's black doing it's thing.  Having said that  we can use `path` their alright.